### PR TITLE
 Fixed: Pressing home key doesn't bring back launcher in some cases.

### PR DESCRIPTION
**Pull request by itsfarseen**
When some apps are opened from the app drawer, pressing the home key didn't bring back the launcher. SoundCloud and Materialistic are the apps I encountered with this behavior.
Android somehow thinks that the activity we are launching is actually part of Silverfish.
The affected apps are not even shown in the Recents list, just like the launcher is not shown.
Pressing back button works fine.

This PR fixes the above issue.

### CODE REVIEW

File: app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java [hunk 1]
```diff
@@ -183,29 +180,6 @@ public void setFragShortcutAddListenerRefreshListener(ShortcutAddListener shortc
 
     //region Events
 
-    private void dropItem(DragEvent dragEvent) {
```
Comment (Lonami) [inline app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java]: Why did you remove this code?

File: app/src/main/java/com/launcher/silverfish/launcher/appdrawer/AppArrayAdapter.java [hunk 2]
```diff
@@ -113,8 +112,7 @@ public void onClick(View view) {
                         e.printStackTrace();
                     }
                 } else {
-                    i.setAction(Intent.ACTION_MAIN);
-                    i.setComponent(new ComponentName(app.packageName.toString(), app.activityName.toString()));
+                    i = mPackageManager.getLaunchIntentForPackage(app.packageName.toString());
```
Comment (Lonami) [inline app/src/main/java/com/launcher/silverfish/launcher/appdrawer/AppArrayAdapter.java:115]: Ah, so that's the way it's done. Cool, thanks.

File: app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java (hunk 1 above)
Comment (itsfarseen) [reply app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java]: Android studio said the method is never used. Shall I add it back?

File: app/src/main/java/com/launcher/silverfish/launcher/appdrawer/AppArrayAdapter.java (hunk 2 above)
Comment (itsfarseen) [reply app/src/main/java/com/launcher/silverfish/launcher/appdrawer/AppArrayAdapter.java:115]: You're welcome :)

File: app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java (hunk 1 above)
Comment (Lonami) [reply app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java]: Yeah, add it back please, if it's really not used maybe that should be another PR, but it's strange it's not (at a glance).

Comment (itsfarseen) [reply app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java]: Roger that :+1:

Comment (itsfarseen) [reply app/src/main/java/com/launcher/silverfish/launcher/LauncherActivity.java]: I removed that commit and force pushed it. Did that work?
